assets/src/edit-story/app/prepublish/guidance/media.js

```diff
@@ -145,15 +145,16 @@ export function videoElementLength(element) {
  * @return {Guidance|undefined} The guidance object for consumption
  */
 export function videoElementOptimized(element = {}) {
-  const idResource = element.resource?.id;
-  const idOrigin = idResource?.toString().split(':')?.[0];
-  const isCoverrMedia = idOrigin === 'media/coverr';
```

Comment:
Let's keep this here. This can be changed in another PR.

Reply:
Agreed, this should be done in a follow up PR.